src/screens/OrgPost/CreatePostModal.tsx

```
@@ -0,0 +1,451 @@
+              {file.type.startsWith('image') ? (
+                <img
+                  src={postformState.addMedia}
+                  data-testid="imagePreview"
+                  alt="Post preview"
+                />
+              ) : (
+                <video controls data-testid="videoPreview">
+                  <source src={postformState.addMedia} type={file.type} />
+                  {/* TODO: add captions (track src) for accessibility */}(
+                  {t('tag')})
+                </video>
```

Biome (2.1.2)

[error] Provide a track for captions when using audio or video elements.

Captions support users with hearing-impairments. They should be a transcription or translation of the dialogue, sound effects, musical cues, and other relevant audio information.

(lint/a11y/useMediaCaption)